### PR TITLE
FUSEDOC-2122 Updated files for help plugin - first try.

### PR DESCRIPTION
**Pull request by TovaCohen**
This request contains most, and maybe all, of the updates that are needed for the help plugin for the 10.0 release. Jane has committed all the content she has worked on. I am still checking the tutorials and some minor updates might be needed in the tutorials. 
I created a test help plugin and I put it into my Developer Studio installation, opened Developer Studio, and the updated help plugin appears to behave correctly. 
We provided a help plugin for testing on July 19. Tomas gave me feedback about content. But I do not know whether or not he tested the plugin itself to make sure that help appears correctly in Developer Studio.

### CODE REVIEW

File: core/plugins/org.fusesource.ide.help/.project [hunk 1]
```diff
@@ -1,39 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<projectDescription>
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.help/.project]: any reasons for having removed the .project and .classpath files? They are useless at runtime but useful during development of Fuse Tooling. is it possible to keep them please?

**Comment by TovaCohen** [comment]
I did not explicitly remove the .project and .classpath files. So something else I did must have caused them to be removed. I'm happy to keep them, but I don't know for sure how to put them back.
I don't want to make a mess -- so can you tell me what I should do?

**Comment by apupier** [comment]
> I did not explicitly remove the .project and .classpath files. So something else I did must have caused them to be removed. I'm happy to keep them, but I don't know for sure how to put them back.
> I don't want to make a mess -- so can you tell me what I should do?

- create a commit on your branch setting back the 2 files
- then do a rebase interactive to squash the new commit with the previous one
- then push force your branch

I can do it my self if you "Allow edits from maintainers" --> https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

**Comment by TovaCohen** [comment]
It looks like Allow Edits From Maintainers is already checked.
If you can fix this for me, that would be great. 
But I also need to understand how to NOT do this again. As I understand it, core/plugins/org.fusesource.ide.help contains .project files and .classpath files and I should make sure that they remain in place when I commit, push, and request a pull. Right?
I have more updates I need to make based on comments from Tomas. So I'd like to see this merge successfully with your fix. And then when I do it again, later today most likely, I'll make sure the .project and .classpath files are in place. I'll be sure to update my branch and fork from upstream. 
Thanks!

**Comment by apupier** [comment]
argh, JBoss Tools already activated the signed-off mandatory.
let me check if I can do it for you.

For the future, in your Eclipse environment:
- click "Preferences > Team > Git > Commit Dialog"
- check "Footers > Insert Signed-off-by"
see https://wiki.eclipse.org/Development_Resources/Contributing_via_Git#Signing_off_on_a_commit for documentation reference

**Comment by apupier** [comment]
testPR

**Comment by apupier** [comment]
> It looks like Allow Edits From Maintainers is already checked.

great

> If you can fix this for me, that would be great.

done

> But I also need to understand how to NOT do this again. As I understand it, core/plugins/org.fusesource.ide.help contains .project files and .classpath files and I should make sure that they remain in place when I commit, push, and request a pull. Right?

exactly

> I have more updates I need to make based on comments from Tomas. So I'd like to see this merge successfully with your fix. And then when I do it again, later today most likely, I'll make sure the .project and .classpath files are in place. I'll be sure to update my branch and fork from upstream.

I will left soon. @bfitzpat  can you have a look to this PR please? As soon as a PR build is done and green this PR can be merged.

**Comment by apupier** [comment]
the build has not started: https://fusesource-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Fuse%20Tooling/job/Fuse-Tooling-pullrequest/
I don't know why. When writing testPR comment it should trigger a build.

**Comment by apupier** [comment]
testPR

**Comment by bfitzpat** [comment]
There is no build associated with this, so I'm not sure what's going on. @nickboldt any ideas on why this particular PR isn't triggering a PR build in Fuse Tools? https://fusesource-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Fuse%20Tooling/job/Fuse-Tooling-pullrequest/

**Comment by nickboldt** [comment]
testPR

**Comment by nickboldt** [comment]
testPR

**Comment by bfitzpat** [comment]
The PR is usually built before the approval is given, so that's a weird change of events.

**Comment by nickboldt** [comment]
OK, so the reason the  PR was not built was that @TovaCohen is not in the admin or whitelist, and then @apupier tried to trigger the job but he too wasn't in the admin list. So I added him and myself, and kicked the PR build with "testPR". https://fusesource-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Fuse%20Tooling/job/Fuse-Tooling-pullrequest/1504

**Comment by bfitzpat** [comment]
@nickboldt thanks for fixing that. Can you add me and @lhein if we're not on those lists as well?

**Comment by bfitzpat** [comment]
Merging per @apupier

**Pull request merged**